attr_report.go

```diff
@@ -0,0 +1,127 @@
+package gtp5gnl
+
+import (
+	"github.com/khirono/go-nl"
+)
+
+const (
+	UR_URRID = iota + 3
+	UR_USAGE_REPORT_TRIGGER
+	UR_URSEQN
+	UR_VOLUME_MEASUREMENT
+	UR_QUERY_URR_REFERENCE
+)
+const (
+	UR_VOLUME_MEASUREMENT_FLAGS = iota + 1
+
+	UR_VOLUME_MEASUREMENT_TOVOL
+	UR_VOLUME_MEASUREMENT_UVOL
+	UR_VOLUME_MEASUREMENT_DVOL
+
+	UR_VOLUME_MEASUREMENT_TOPACKET
+	UR_VOLUME_MEASUREMENT_UPACKET
+	UR_VOLUME_MEASUREMENT_DPACKET
+)
+
+type USAReport struct {
+	URRID          uint32
```

Comment:
How does UPF notify usage report to an associated session?
I think USAReport needs to have the SEID field.

Reply:
Currently, gtp5g will use unix_sock_send() to send a message containing SEID and reports to UPF. The UPF can then decode the SEID and find the associate session. Do I need to integrate the SEID into the report struct?

Reply:
That's fine. I confirmed that SEID is notified besides this structure.